simulation/traffic_simulator/include/traffic_simulator/api/api.hpp

```diff
@@ -74,7 +75,21 @@ class API
       rclcpp::PublisherOptionsWithAllocator<AllocatorT>())),
     debug_marker_pub_(rclcpp::create_publisher<visualization_msgs::msg::MarkerArray>(
       node, "debug_marker", rclcpp::QoS(100), rclcpp::PublisherOptionsWithAllocator<AllocatorT>())),
-    clock_(std::forward<decltype(xs)>(xs)...),
+    real_time_factor_subscriber(rclcpp::create_subscription<std_msgs::msg::Float64>(
+      node, "/real_time_factor", rclcpp::QoS(rclcpp::KeepLast(1)).best_effort(),
+      [this](const std_msgs::msg::Float64 & message) {
+        /*
+          Pausing the simulation by setting the realtime_factor_ value to 0 is not supported and causes the simulation crash.
+          For that reason, before performing the action, it needs to be ensured that the incoming request data is a positive number.
+        */
```

Comment:
Please use doxygen @note command.

Reply:
@hakuturu583 I have just updated the comment.